cfpb/design-manual
# Add animated icons examples to the Design Manual documentation

We currently don't document the animated icons despite them being a part of Capital Framework for some time.

Animating minicons has been available in Capital Framework for some time
but never documented. This change adds that missing documentation.

## Additions

- Added animated icons documentation based on @caheberer GDoc.
- Added examples of animated icons.

## Testing

- run `npm update` to make sure you have the latest from Capital Framework
- run `grunt build`, if you get an error run `grunt build -f`. There's a bug in the linting file.
- run `npm start` and navigate to `http://localhost:4000/brand-guidelines/minicons.html#animation`

## Review

- @caheberer 
- @Scotchester 
- @sebworks

[Preview this PR without the whitespace changes](?w=0)

## Screenshots

![screen shot 2017-04-27 at 5 26 00 pm](https://cloud.githubusercontent.com/assets/1280430/25506899/99a26702-2b6e-11e7-93f9-b6a58f2a1f59.png)

## Checklist
